data/templates/centos-ks

```diff
@@ -126,6 +130,8 @@ echo -e "NETWORKING=yes\nHOSTNAME=<%=hostname%>.<%=domain%>" > /etc/sysconfig/ne
       <% if( undefined != n.ipv6.vlanId ) { %>
         <% n.ipv6.vlanId.forEach(function(vid) { %>
           echo "Configuring vlan <%=vid%> on <%=n.device%>"
+          sed -i '/^BOOTPROTO=/d' /etc/sysconfig/network-scripts/ifcfg-<%=n.device%>
```

Comment:
Same comment here, should be /etc/sysconfig/network-scripts/ifcfg-<%=n.device%>.<%=vid%>